e2e/test/iothub/messaging/MessageSendFaultInjectionTests.cs

```diff
@@ -6,21 +6,22 @@
 using System.Threading.Tasks;
 using Microsoft.Azure.Devices.Client;
 using Microsoft.Azure.Devices.Client.Exceptions;
+using Microsoft.Azure.Devices.E2ETests.Helpers;
+using Microsoft.Azure.Devices.E2ETests.Helpers.Templates;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace Microsoft.Azure.Devices.E2ETests
+namespace Microsoft.Azure.Devices.E2ETests.Messaging
 {
     [TestClass]
     [TestCategory("E2E")]
     [TestCategory("IoTHub")]
     [TestCategory("FaultInjection")]
-    public partial class MessageSendFaultInjectionTests : IDisposable
+    public partial class MessageSendFaultInjectionTests : E2EMsTestBase
     {
         private readonly string _devicePrefix = $"E2E_{nameof(MessageSendFaultInjectionTests)}_";
 
 #pragma warning disable CA1823
-        private static TestLogger _log = TestLogger.GetInstance();
-        private readonly ConsoleEventListener _listener = TestConfig.StartEventListener();
+        private static TestLogger s_log = TestLogger.GetInstance();
```

Comment:
I believe there are lots of this. Would be great if you can just update to using the base logger. But if it is too much work let me know and we can do it as a separate task.

Reply:
I've removed all direct references to `TestLogger`, and added references tp `MsTestLogger`, instead.

Reply:
This is turning out to be more of renamings; you can take this up if you are already working on that code, i'll revert my changes :D

Reply:
Probably just extend from base class but don't do any other renaming. Will be easier to do it in the following PR.

Reply:
yes, agreed.